functions/gcs/index.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var gcloud = require('gcloud');
+var readline = require('readline');
+
+module.exports = {
```

Comment:
Does it make sense separate `wordCount` into a top-level function and then do `exports.wordCount = wordCount`? Or is it accepted style to do it this way for single-function modules?